src/main/index.js

```diff
@@ -0,0 +1,6 @@
+import './assets/scss/main.scss';
+import { initAll } from 'govuk-frontend';
+import { initAdoptionModules } from './assets/scripts/main';
+
+initAll();
+initAdoptionModules();
```

Comment:
No adoption modules yet, just placeholder for such a time we do include our own. Perhaps in post code lookup.